_guidelines-v4/04-cmn/03-cmnadditionalmeasurecontent/06-cmninstr/02-cmnharp.md

```diff
@@ -6,10 +6,10 @@ version: "v4"
 
 Modern harps have seven pedals which allow adjustment of their strings to different pitches. The settings for these pedals occur at the beginning of the harp notation and/or whenever it is necessary to change the harp's tuning. These settings may be rendered using letter pitches (in the order of the pedals from left to right) or in a diagrammatic fashion, such as the form invented by Carlos Salzedo.
```

Comment:
can we perhaps have a link to Salzedo's model? According to [Wikipedia](https://en.wikipedia.org/wiki/Carlos_Salzedo#Death_and_legacy), "_no single source compiled the effects and their notations_", but maybe there's something to be found in NewGrove or elsewhere? (sorry, no access to such ressources on my train…)

Reply:
At least, we should have an image of that notation – should be something like the stuff in this screenshot: 
<img width="499" alt="grafik" src="https://user-images.githubusercontent.com/2078563/67807695-b369a280-fa6b-11e9-9c69-9d74087145fb.png"> (Jindřich Feld, _Sonatine pour Flûte et Harpe_, 1986)

Reply:
Here are the SCORE graphic symbols for harp pedals, which are probably more standardized than Salzedo's model (at least it is easier for me to interpret the symbols in this graphic syntax):

<img width="633" alt="Screen Shot 2019-10-29 at 2 08 49 PM" src="https://user-images.githubusercontent.com/3487289/67809392-da1cde80-fa55-11e9-9f03-0e7e4b30a020.png">

This is scanned from the SCORE 3.0 reference manual (published around 1992).

Reply:
I'm not even perfectly sure if the scan I came up with is from Salzedo or not – maybe your proposal is from him :-) I just think we should try to give an example here (as we should in many other places), but this could happen after merging, I think

Reply:
Here is a link showing how to create graphic harp pedal symbols for MuseScore, Finale and Sibelius, all of which follow SCORE's graphical model:

https://www.scoringnotes.com/tips/create-harp-diagrams-in-sibelius-finale-and-musescore

Reply:
see d29b566